kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -377,6 +378,17 @@ export class BrowserInterface {
 
     DCL.gifProcessor.ProcessGIF(data)
   }
+
+  public async FetchBalanceOfMANA() {
+    const identity = getIdentity()
+
+    if (!identity.hasConnectedWeb3) {
+      return
+    }
+
+    const balance = await getERC20Balance(identity.address, decentralandConfigurations.paymentTokens.MANA)
+    unityInterface.UpdateBalanceOfMANA(`${balance}`)
```

Comment:
Can we caché the previous balance to avoid messaging the client when the balance hasn't changed? (escaping inside that `FetchBalanceOfMANA()` when that condition happens.

Because that's gonna be the most normal case mostly

Reply:
Yeap, good point there